src/styles/blocks/product.scss

```diff
@@ -0,0 +1,95 @@
+.product {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin-bottom: 40px;
+
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__description {
+    margin: 0 0 4px;
+    height: 36px;
+
+    align-self: center;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__reviews-wrapper {
+    margin: 0 0 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-wrapper {
+    margin: 0 0 16px;
+
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    margin: 0;
+
+    font-size: 12px;
+    line-height: $line-height;
+  }
+
+  &__price-value {
+    margin: 0;
+
+    font-weight: 700;
+    font-size: 16px;
+    line-height: $line-height;
+  }
```

Comment:
You can reset margin in your reset.scss file for required tags to avoid repetition.

Reply:
Done